dbbackup/management/commands/dbrestore.py

```diff
@@ -77,7 +77,8 @@ def restore_backup(self):
         if not self.filepath:
             self.log("  Finding latest backup", 1)
             filepaths = self.storage.list_directory()
-            filepaths = [f for f in filepaths if f.endswith('.' + self.backup_extension)]
+            # TODO: It is a bad filter
+            # filepaths = [f for f in filepaths if f.endswith('.' + self.backup_extension)]
```

Comment:
I'm seeing there is 'dbbackup.management.commands.dbbackup.Command.list_backups` with a TODO comment.
Maybe it was created for this?